sourcecode/hub/routes/web.php

```diff
@@ -79,12 +79,18 @@
     Route::get('/register', 'register')->name('register');
     Route::post('/register', 'store');
 
+    Route::get('/forgot-password', 'showForgotPasswordForm')->name('forgot-password');
+    Route::post('/forgot-password', 'sendResetLink')->name('forgot-password-send');
+
+    Route::get('/reset-password/{token}/{email}', 'showResetPasswordForm')->name('reset-password');
+    Route::post('/reset-password/{token}/{email}', 'resetPassword')->name('reset-password-update');
+
     Route::middleware('auth:web')->group(function () {
         Route::get('/preferences', 'preferences')->name('user.preferences');
         Route::post('/preferences', 'savePreferences')->name('user.save-preferences');
 
         Route::get('/my-account', 'myAccount')->name('user.my-account');
-        Route::post('/save-name', 'saveName')->name('user.save-name');
+        Route::post('/updateAccount', 'updateAccount')->name('user.update-account');
```

Comment:
This route breaks with the established naming convention. It should be `/update-account`.